### PR TITLE
Frontend : CSS Library in document form

### PR DESCRIPTION
**Pull request by Pranshu321**
<hr>

## Description 📜

In this, I have explained the important CSS libraries and included a clear and crisp explanation.
Issue Number Closed :- #7560

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev HTML/CSS/JS

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Web_Development/FrontEnd/CSS_Libraries.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+- Cursor
+- Visibility
+- Opacity
+-  Float
+- Position
+
+6. **Font Awesome :** This is a CSS library which have vector icons and logos these are easily customize for your web designs. These icons are in multiple variants, as well, with each with good asset like light, regular, or solid styles . 
+-  It is extremely light as well as being easy to install and use
+- Font Awesome is as simple as cutting and pasting lines into your code.
+
```
Comment (2024-SANDHYA) [inline Web_Development/FrontEnd/CSS_Libraries.md:42]: Font awesome can't be said a CSS Library, but icons.

File: Web_Development/FrontEnd/CSS_Libraries.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+7. **Ionic :**  This have the features of a full array of CSS utilities and components and featuring everything from Headers to Buttons to Customized Text all things are possible with ionic .
+- It also allows for cross-platform development .
+- It have native integration with most popular development environments .
+- It also make prototyping easy, quick, and painless .
```
Comment (2024-SANDHYA) [inline Web_Development/FrontEnd/CSS_Libraries.md:46]: Plagiarized content here.

File: Web_Development/FrontEnd/CSS_Libraries.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
+# CSS Libraries
+CSS Libraries are the assets which help us to made the website or any other thing from scratch and allows you to design your flows . \
```
Comment (2024-SANDHYA) [inline Web_Development/FrontEnd/CSS_Libraries.md:2]: Remove \

File: Web_Development/FrontEnd/CSS_Libraries.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+### Important Points :
+1. Full control in your hand .
+2. Allows you design your methods .
+3. More flexible than the frameworks .
+
+## Most used Libraries 
+1. **Animate css :** This library help us to implement animations in our website and generate interest and favorable impressions of your website . 
+- CSS animations that can be implemented across multiple browsers like safari,chrome,opera and more .
```
Comment (2024-SANDHYA) [inline Web_Development/FrontEnd/CSS_Libraries.md:10]: Check the alignment

**Review by 2024-SANDHYA — CHANGES_REQUESTED** [review]
No reference links mentioned too.

**Review by 2024-SANDHYA — CHANGES_REQUESTED** [review]
Reference links missing.

**Comment by Pranshu321** [comment]
> Reference links missing.

Reference website added

**Review by 2024-SANDHYA — APPROVED** [review]

**Pull request merged**